packages/kit/src/core/generate_manifest/index.js

```diff
@@ -41,10 +42,13 @@ export function generate_manifest({ build_data, relative_path, routes, format =
 	});
 
 	/** @type {(path: string) => string} */
-	const importer =
+	const load =
```

Comment:
I personally liked `importer` a bit more to avoid conflating with the page `load`